readme.md

```diff
@@ -227,7 +227,9 @@ While annotations are not specified by IIIF they are an important enabling techn
 Content Management Systems (CMS) modules that implement or leverage the IIIF APIs.
 
 - [IIIF Image Field](https://www.drupal.org/sandbox/sdellis/2421047) - Drupal 7 module that provides an easy way to add IIIF Images to content types, and configure their display. Supports Image API versions 1.0 or 2.0.
-- [UniversalViewer Omeka S Plugin](https://omeka.org/s/modules/UniversalViewer/) - Universal Viewer is a module for Omeka S that integrates UniversalViewer, a unified online player for any files, so it can display books, images, maps, audio, movies, pdf, 3D, and anything else as long as the appropriate extension is installed. Rotation, zoom, inside search, etc. may be managed too. It uses the resources of any IIIF compliant server.
+- [UniversalViewer Omeka S Module](https://omeka.org/s/modules/UniversalViewer/) - Universal Viewer is a module for Omeka S that integrates UniversalViewer, a unified online player for any files, so it can display books, images, maps, audio, movies, pdf, 3D, and anything else as long as the appropriate extension is installed. Rotation, zoom, inside search, etc. may be managed too. It uses the resources of any IIIF compliant server.
+- [IIIF Server Omeka S Module](https://omeka.org/s/modules/IiifServer/) - IIIF Server is a module for Omeka S that integrates the IIIF specifications (level 2) and a simple image server (similar to a basic IIP Image) to allow to process and share instantly images of any size and medias (pdf, audio, video, 3D . . .) in the desired formats. Any widget that supports IIIF level 2 can use this plugin. Rotation, zoom, inside search, etc. may be managed. Dynamic lists of records may be created, for example for browse pages.
+- [Mirador Viewer Omeka S Module](https://github.com/Daniel-KM/Omeka-S-module-Mirador) - Mirador Viewer is a module for Omeka S that integrates Mirador, an advanced online viewer for images, so it can display books, images, maps, etc. via the IIIF standard. Common plugins are included. It uses the resources of any IIIF compliant server. This module supports IIIF level 2.
```

Comment:
Module for Omeka S which integrates Mirador.

Reply:
Shortened description along the lines of @jronallo feedback.